README.md

```diff
@@ -94,6 +94,53 @@ jobs:
         arch: ${{ matrix.arch }}
 ```
 
+#### Building for Automated Tests
+
+As described in the [Inputs](#inputs) documentation, specifying `run-tests: true` will amend the Flatpak manifest to enable Network and X11 access automatically. Any other changes to the manifest must be made manually, such as building the tests (e.g. `-Dtests=true`) or any other options (e.g. `--buildtype=debugoptimized`).
+
+Most developers will want to run tests on pull requests, before merging into the main branch of a repository. To ensure your tests are being run on the pull request, you should set the `sources` entry for your project to `"type": "dir"` with the `path` key relative to the manifest's location in the repository.
+
```

Comment:
That is not specific to running tests by the way

Reply:
Good point, maybe I should amend to to say "To ensure your manifest is building the correct code, ...`?